model/host/host.go

```diff
@@ -1721,3 +1713,14 @@ func (h *Host) SetTags() error {
 		},
 	)
 }
+
+// SetInstanceType updates the host's instance type in the database.
+func (h *Host) SetInstanceType(instanceType string) error {
```

Comment:
I worry about putting this in the host document as a top level field, because it ends up being provider specific and difficult to validate. I'm not sure though.

Reply:
Offline: yes it's icky because not every host has an instance type (ie. docker, static) but putting it in the provider settings would be difficult to set and probably not worth the hassle.